### PR TITLE
Calculates size_remaining from size and size_matched when not set

### PR DESCRIPTION
**Pull request by lunswor**

### CODE REVIEW

File: flumine/order/order.py [hunk 1]
```diff
@@ -391,7 +391,10 @@ def size_remaining(self):
         try:
             return self.current_order.size_remaining or 0.0
         except AttributeError:
-            return 0.0
+            if self.size_matched > 0:
```
Comment (liampauling) [inline flumine/order/order.py:394]: size_matched could be None? I think?

**Review by liampauling — CHANGES_REQUESTED** [review]

File: flumine/order/order.py (hunk 1 above)
Comment (lunswor) [reply flumine/order/order.py:394]: I didn't think so based on https://github.com/liampauling/flumine/blob/master/flumine/order/order.py#L385 handling the potential None

Comment (liampauling) [reply flumine/order/order.py:394]: Good point, can you a small comment on this line as to why we do this? (placeResponse does not include sizeRemaining)

Comment (liampauling) [reply flumine/order/order.py]: Thanks, through some strategic print statements I was able to replicate your bug on v0.14.3 where size matched and size remaining both returned 0 when the order was in flight, this branch has fixed this 👍

**Pull request merged**